Phonebook.Frontend/src/app/pages/page-information/page-information.component.html

```diff
@@ -1,82 +1,102 @@
-<mat-card class="pb-margin-20">
-  <mat-card-header>
-    <mat-card-title>
-      <h1 i18n="PageInformationComponent|Title of the page information page@@PageInformationComponentTitle">
-        The Phonebook
-      </h1>
-    </mat-card-title>
-    <mat-card-subtitle
-      *ngIf="isPreview"
-      i18n="Page-informationComponent|Disclaimer beneath title@@PageInformationComponentDisclaimer"
-    >
+<div class="pb-margin-left-right-resposive">
+  <div class="header pb-margin-top-20">
+    <h1 i18n="PageInformationComponent|Title of the page information@@PageInformationComponentTitle">
+      Phonebook
+    </h1>
+    <p *ngIf="isPreview" i18n="Page-informationComponent|Disclaimer beneath title@@PageInformationComponentDisclaimer">
       This is a Preview application. We are doing our best not to break anything, but we can't guarantee. Thats why all
       Features are still subject to change without further notice and your data may get lost.
-    </mat-card-subtitle>
-  </mat-card-header>
-
-  <mat-card-content>
-    <h2 i18n="Page-informationComponent|Subtitle for Contact and Feedback@@PageInformationComponentSubTitleFeedback">
-      Contact & Feedback
-    </h2>
-    <mat-list i18n="Page-informationComponent|Feedback Section@@PageInformationComponentSectionFeedback">
-      <mat-list-item>
-        <p>
+    </p>
+  </div>
+  <div class="pb-flex pb-justify-center">
+    <!-- contact and feedback -->
+    <mat-card class="small-card pb-border-color-primary">
+      <mat-card-header>
+        <mat-card-title><mat-icon [color]="'primary'">sentiment_satisfied_alt</mat-icon></mat-card-title>
```

Comment:
By the way you can just write:
```suggestion
        <mat-card-title><mat-icon color="primary">sentiment_satisfied_alt</mat-icon></mat-card-title>
```